Mathlib/Combinatorics/SimpleGraph/Walk.lean

```diff
@@ -1032,13 +1062,19 @@ theorem mem_support_iff_exists_getVert {u v w : V} {p : G.Walk v w} :
         rw [@nil_iff_length_eq]
         have : 1 ≤ p.length := by omega
         exact Nat.not_eq_zero_of_lt this
-      rw [← tail_support_eq_support_tail _ hnp]
+
```

Comment:
Please don't include blank lines in proofs. If you feel like the proof could do with some more control flow, please add comments instead of blank lines

Reply:
Fixed:)